File: tests/fixtures/coding-eval/go/sample_test.go
package sample

import "testing"

func TestDivide(t *testing.T) {
	if Divide(5, 2) != 3 {
		t.Fatal("integer division must round up")
	}
}

func TestLabel(t *testing.T) {
	if Label("alpha") != "entry:alpha" {
		t.Fatal("unexpected label")
	}
}

func TestNormalize(t *testing.T) {
	if NormalizeName("  Alpha   BETA ") != "alpha beta" {
		t.Fatal("unexpected normalized name")
	}
}

func TestActive(t *testing.T) {
	if !Active(Status{State: "active"}) || Active(Status{State: "paused"}) {
		t.Fatal("unexpected activity state")
	}
}

File: tests/fixtures/coding-eval/zig/sample_test.zig
const std = @import("std");
const sample = @import("sample.zig");

test "divide" {
    try std.testing.expectEqual(@as(u32, 3), try sample.divide(5, 2));
    try std.testing.expectError(error.DivisionByZero, sample.divide(1, 0));
}

test "label" {
    var output: [64]u8 = undefined;
    try std.testing.expectEqualStrings("entry:alpha", try sample.label("alpha", &output));
}

test "normalize" {
    var output: [64]u8 = undefined;
    try std.testing.expectEqualStrings(
        "alpha beta",
        try sample.normalizeName("  Alpha   BETA ", &output),
    );
}

test "active" {
    try std.testing.expect(sample.active(.{ .state = "active" }));
    try std.testing.expect(!sample.active(.{ .state = "paused" }));
}
